src/styles/blocks/stars--5.css

```diff
@@ -0,0 +1,44 @@
+.stars--5 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--5 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
+}
+
+.stars--5 .stars__star:nth-child(2) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
+}
+
+.stars--5 .stars__star:nth-child(3) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
+}
+
+.stars--5 .stars__star:nth-child(4) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
+}
+
+.stars--5 .stars__star:nth-child(5) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 4px;
+}
```

Comment:
To improve the maintainability of the code, consider using a single CSS rule for the `.stars__star` elements that share the same styles. You can then override the `margin-left` for the first child if it needs to be different.